server/features/base_report.feature

```
@@ -317,26 +317,26 @@ Feature: Base Analytics Report Service
         ]
         """
         When we get "/analytics_test_report?params={"must": {}, "size": 2, "page": 1, "sort": [{"slugline": "asc"}]}&aggs=0&max_results=2"
-        Then we get list with 6 items
+        Then we get list with 2 items
```

Something is wrong here, as the list should return 6 items. Rather than make the test pass, it might be best to let it fail so we know it doesn't work and can work on the fix

---

yeah I know but if you look at it carefully, it seems like the test was wrong. Notice that it is actually testing against two items (which is what's coming from the response and seems correct)

```python
"_items": [{"_id": "archive1"}, {"_id": "archive2"}],
```

---

But the `we get list with` step checks against the [_meta.total](https://github.com/superdesk/superdesk-core/blob/async/superdesk/tests/steps.py#L1347) attribute, which should contain the number of total items for the query, not the number of items in that particular page

---

yeah, that makes senses. Although the test it's a bit confusing. "we get a list with X items". The response has a list attributed called `_items` which contains 2 items instead of 6. Perhaps I would change this to something more meaningful.